apps/app/pages/admin/lectures/presences/index.tsx

```diff
@@ -0,0 +1,282 @@
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { Button, Row, Select, Space, Table, Typography } from "antd";
+import { CloseOutlined, EditOutlined, SaveOutlined } from "@ant-design/icons";
+
+import {
+  getNinjaEvents,
+  listEvents,
+  listLectures,
+  listMentors,
+} from "bokkenjs";
+import * as api from "bokkenjs";
+import { notifyError, notifyInfo } from "~/components/Notification";
+import AppLayout from "~/layouts/AppLayout";
+import { set } from "lodash-es";
+const { Title } = Typography;
+
+type Event = {
+  id: string;
+  title: string;
+};
+
+type Lecture = {
+  id: string;
+  event: Event;
+  attendance: string;
+};
+
+export default function Presences() {
+  const [editButtonVisible, setEditButtonVisible] = useState(true);
+  const [saveButtonVisible, setSaveButtonVisible] = useState(false);
+  const [cancelButtonVisible, setCancelButtonVisible] = useState(false);
+  const [data, setData] = useState<any[]>([]);
+  const [originalData, setOriginalData] = useState<any[]>([]);
+  const [events, setEvents] = useState<Event[]>([]);
+  const [selectedEvent, setSelectedEvent] = useState<string>("");
+  const [lectures, setLectures] = useState<Lecture[]>([]);
+  const [selectedLectures, setSelectedLectures] = useState<Lecture[]>([]);
+  const [locations, setLocations] = useState<any[]>([]);
+  const router = useRouter();
+
+  const onFinish = (values: any, lectureId: string) => {
+    api
+
+      .updateLecture(lectureId, values)
+      .then(() => {
+        notifyInfo("Os dados da sessão foram atualizados com sucesso", "");
+      })
+      .catch((error) => {
+        notifyError(
+          "Ocorreu um erro",
+          "Não foi possível atualizar os dados da sessão"
+        );
+      });
+  };
+
+  useEffect(() => {
+    api
+      .listEvents()
+      .then((response: any) => setEvents(response.data))
+      .catch(() => {});
```

Comment:
Add a notification for this error. The worst thing you can do is having the request fail silently.